src/cart/crt_swim.c

```diff
@@ -332,16 +331,16 @@ static int crt_swim_send_message(struct swim_context *ctx, swim_id_t to,
 
 	rc = crt_req_create(crt_ctx, &ep, opc, &rpc_req);
 	if (rc) {
-		D_ERROR("crt_req_create() failed "DF_RC"\n", DP_RC(rc));
+		D_ERROR("crt_req_create(): "DF_RC"\n", DP_RC(rc));
```

Comment:
We need to settle on a standard format for this as there's a lot of variation and changes in format and it's not always in the same direction.

Reply:
I'm fine with any...